ChangeLog.org

```diff
@@ -1,5 +1,44 @@
 * Changelog for Juvix
 
+** 0.1.1.0
+ - _Summary_:
+   + new syntax for effect handlers in Juvix.
+
+ - _Main contributions_:
+   + adds `effect` and `handler` syntaxes (also `_ via _`);
+   + removes do notation as `Prelude.»=`.
+
+*** Added Syntaxes
+#+BEGIN_SRC ocaml
+effect Print where
+  | op print : string -> ()
+  | return   : x -> string
+#+END_SRC
+A new effect Print is defined, and its interface signature. All
+effects must have a `return` clause, and hopefully operation
+signatures as well (an effect without operations is just a pure
+computation).
+
+#+BEGIN_SRC ocaml
+handler print where
```

Comment:
Shouldn't `print` start with a capital here?

Reply:
handlers are lowercase, while effects are uppercase (convention, not enforced). you can also name handlers whatever you want regarless of the effect you handling

Reply:
So the connection between an effect and it's handler is the fact that the handler has a operation with the correct name and type available?

Reply:
exactly, same operation name and type. you can even handle multiple different effects within a single handler if you want to, or not handle some operations of an effect... as long as some other handler handles that operation you omitted -- otherwise it's a handling error.